nemo/collections/asr/modules/audio_preprocessing.py

```diff
@@ -776,6 +792,14 @@ def forward(
         with torch.cuda.amp.autocast(enabled=False):
             output = self.stft(input.float())
 
+            if self.magnitude_power != 1:
```

Comment:
curious to understand: is this a learnable module? I doesn;t seem so, hence why not mention `no_grad` to forward here?